src/utils/childrenIsAnImage.ts

```diff
@@ -0,0 +1,3 @@
+export const childrenIsAnImage = (children: any) => {
+  return typeof children[0] === 'object' && children[0].props.hasOwnProperty('src');
```

Comment:
Just thinking, would there be anything else that may contain a `src` property that could cause issues here? I can't think of anything from markdown, just making sure.

Reply:
Don't think so, we have embedded videos on the docs which are also children of p tags, but isntead of `src` they have `href`.